docs/development-environment-cli-support.md

```diff
@@ -8,6 +8,15 @@ tools/check-development-environment.sh
 
 ## Topics
 
+### Command is available: jetpack
```

Comment:
This doesn't make a "jetpack" command available. It does make "yarn jetpack" available.

Reply:
To confirm, after running `yarn install`, you aren't able to run `jetpack` directly? `yarn install` should be adding a symlink to `/usr/local/bin/jetpack` for this upon install.


![2021-01-25 at 10 56 AM](https://user-images.githubusercontent.com/88897/105737770-f549f880-5efb-11eb-8221-223a70dade7a.png)

Reply:
same here. `jetpack` command is not available (`/usr/local/bin/jetpack` not found)

Reply:
That's correct. It doesn't even give an error about trying to mess with /usr/local/bin, probably because my normal user doesn't have write access to /usr/local/bin so it doesn't even try.

Nor would I want `yarn install` in some repo to be messing with my /usr/local/bin in the first place. Eew.

Reply:
@brbrr In your case, does it work in a new shell?

@anomiex In effect, you're running as `yarn install --no-bin-links`

Reply:
> does it work in a new shell?

Nope, as I mentioned, `jetpack` not gets symlinked to `/usr/local/bin`. And I kinda agree with Brad, I don't think it is expected for an npm package to be accessible in PATH unless it is installed globally

Reply:
> @anomiex In effect, you're running as yarn install --no-bin-links

Not really. That doesn't create `node_modules/.bin/` either, which would break lots of yarn things.

Reply:
The big difference between `jetpack` and `yarn jetpack` is `yarn jetpack` only works at the repo level. If you run that within the jetpack plugin, it won't work.

The expected behavior of the `bin` section of package.json is to make that script available by adding it to a location within a `PATH` folder. 

Reading the [Yarn classic source code](https://github.com/yarnpkg/yarn/blob/a4708b29ac74df97bac45365cba4f1d62537ceb7/src/cli/commands/global.js#L84), it looks like they may fallback to a `.yarn` dir in your home directory. Do you have a `.yarn/bin` dir that has Jetpack in it? Does the `jetpack` command work if you include `.yarn/bin` in your PATH?

Reply:
I have `~/.yarn/bin`, but it doesn't have `jetpack` in it.

I note the code you're looking at is for the `yarn global` command, where the user is specifically installing something globally.

Reply:
It was `yarn link`. When you run that from within tools/cli, it adds a global symlink to the local version. When you run it from within the repo root (`yarn link jetpack-cli` afterwards), it'll tell yarn to not use the node_modules that it installed of it, but use the global symlink version instead.

The location of that symlink seems driven by https://github.com/yarnpkg/yarn/blob/a4708b29ac74df97bac45365cba4f1d62537ceb7/src/config.js#L470

I'm curious if you run `yarn link` within the tools/cli folder, if it'll work for you? With me adding the `jetpack cli init` command, this may just work now without moving it.

Reply:
Going to merge and we can follow-up if still an issue. I've soften the check in the dev env check script for now.

Reply:
Looks like `yarn link` does create a link in `~/.yarn/bin`. That doesn't seem like the behavior I'd expect here. Apparently `yarn link` is for situations where you're developing a library and want to easily test unrelated projects that use it, rather than the monorepo use case we have here.

Reply:
@anomiex It works both ways. We'd need to do that to make changes to the cli and immediately be able to use them and, in effect, it installs it globally too.

We can be more explicit about requiring it to be installed globally, but I think, to ensure we can make changes to it live, we'd need to link it install of installing it anyhow.

The outstanding bit would be clarifying about ~/yarn/.bin in the PATH.